components/catalogs/front/src/components/ProductMapping/components/SourceSelection/SourceSelection.tsx

```diff
@@ -0,0 +1,46 @@
+import React, {FC} from 'react';
+import {Source} from '../../models/Source';
+import {Target} from '../../models/Target';
+import {SourceErrors} from '../../models/SourceErrors';
+import {useAttribute} from '../../../../hooks/useAttribute';
+import {Attribute} from '../../../../models/Attribute';
+import {SectionTitle, Tag} from 'akeneo-design-system';
```

Comment:
```suggestion
import {SectionTitle as SectionSubtitle, Tag} from 'akeneo-design-system';
```
to be consistent with `SourceParameters` ?
WDYT ? Is it over-engineered ?

Reply:
SectionSubtitle in SourceParameters is a component that contains multiple components (that uses SectionTitle)
Here SectionTitle is used to display a single string so it would be strange to alias it that way don't you think?

Reply:
Yes that's ok for me, just wondering if we can have something cleaner to avoid confusion with the 2 levels of title. What do you think about extracting the SectionSubtitle as a standalone component with 2 props (number and title) and using it in both places?

Reply:
done [here](https://github.com/akeneo/pim-community-dev/pull/19285/commits/21e67b26c71a06e9ccb45859f0efe2a7e82da40d)